bluesky/run_engine.py

```diff
@@ -2348,7 +2358,8 @@ async def _stop(self, msg):
 
             Msg('stop', obj)
         """
-        return msg.obj.stop()  # nominally, this returns None
+        obj = check_supports(msg.obj, Stoppable)
+        return await maybe_await(obj.stop())  # nominally, this returns None
```

Comment:
Easier to `await` then do the dance with an `Event` and a `Status` here, as with `pause` and `resume`